### ISSUE TITLE
Multitenant vmalert external label support per tenant

### ISSUE THREAD

**Issue by arramos84** [VictoriaMetrics/VictoriaMetrics#1471]
**Is your feature request related to a problem? Please describe.**

The new multitenant vmalert doesn't seem to support external labels on a per tenant basis. This would be valuable and practical as different tenants may require different labels.

**Describe the solution you'd like**

Added support for external labels to multitenant vmalert

**Describe alternatives you've considered**
Injecting the label via custom code that generates the rules.

**Comment by f41gh7** [comment]
I think, that it should be defined per configuration group. For example:
```yaml
groups:
- name: rules_for_tenant_123
  tenant: "123"
  extra_labels:
     label: value
  rules:
    # Rules for accountID=123
- name: rules_for_tenant_125
  tenant: "125"
  extra_labels:
     label-125: value-125
  rules:
    # Rules for accountID=125

```

@hagen1778  WDYT?

**Comment by tenmozes** [comment]
It looks good let's define on group level. Let's also take into account only group external labels and don't merge them with labels provided by flag

**Comment by hagen1778** [comment]
To avoid confusion between `extra` and `external` labels, should we name it just as `labels` and remain consistent with `labels` field on the rule level?

**Comment by hagen1778** [comment]
Hi @arramos84! https://github.com/VictoriaMetrics/VictoriaMetrics/pull/1485 has been merged and will be available in the next release. Thanks!

**Comment by valyala** [comment]
The ability to set per-group labels via `labels` section is available in `vmalert` starting from [v1.65.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). See [the docs on groups section](https://docs.victoriametrics.com/vmalert.html#groups).

Closing the feature request as done.
